jena-arq/Grammar/master.jj

```diff
@@ -100,6 +100,38 @@ import org.apache.jena.sparql.core.Quad ;
 public class CLASS extends PARSERBASE
 {
     boolean allowAggregatesInExpressions = false ;
+
+    public static void main(String args[]) {
+        while (true) {
```

Comment:
Not sure where else it could go. I copied (shamelessly) from an example I found online while re-reading about JavaCC. With this main method, you can run the grammar in Eclipse, and in the Eclipse Console it will be waiting for a String+LFLF (two break lines IIRC).

Then it will use the grammar to parse the string and will output the QueryUnit. I found it useful for reviewing the changes without running some extra class with a main method, or Fuseki.

What do you think? I'm OK with removing it, or moving it somewhere else. Just don't know where else it could go :-)